server/migrations/20200521003201-create_table_test_issues.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('test_issue', {
+            id: {
+                type: Sequelize.INTEGER,
+                primaryKey: true,
+                autoIncrement: true
+            },
+            test_id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'test',
+                    key: 'id'
+                }
+            },
+            run_id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                references: {
+                    model: 'run',
+                    key: 'id'
+                }
+            },
+            issue_number: {
+                type: Sequelize.INTEGER,
+                allowNull: false
+            }
```

Comment:
I think you may be missing the `open` boolean column?

Reply:
I just got an error about this @s3ththompson , I'll push a commit...